website/docs/authenticating_via_azure_cli.html.markdown

```diff
@@ -9,77 +9,81 @@ description: |-
+When authenticating via the Azure CLI, Terraform will automatically connect to the Default Subscription. The Default Subscription can be changed via using the Azure CLI and is documented below.
 
 ## Configuring the Azure CLI
 
-~> **Note:** There are multiple versions of the Azure CLI - the latest version is known as [the Azure CLI 2.0 (Python)](https://github.com/Azure/azure-cli) and [the older Azure CLI (Node.JS)](https://github.com/Azure/azure-xplat-cli). While Terraform currently supports both - we highly recommend users upgrade to the Azure CLI 2.0 (Python) if possible.
+~> **Note:** There are multiple versions of the Azure CLI. The latest version is known as [the Azure CLI 2.0 (Python)](https://github.com/Azure/azure-cli) and [the older Azure CLI (Node.JS)](https://github.com/Azure/azure-xplat-cli). While Terraform currently supports both, we highly recommend users upgrade to the Azure CLI 2.0 (Python), if possible.
 
 This guide assumes that you have [the Azure CLI 2.0 (Python)](https://github.com/Azure/azure-cli) installed.
 
-~> **Note:** If you're using the **China**, **German** or **Government** Azure Clouds - you'll need to first configure the Azure CLI to work with that Cloud.  You can do this by running:
+~> **Note:** If you're using the **China**, **German** or **Government** Azure Clouds, you'll need to first configure the Azure CLI to work with that Cloud. You can do this by running:
 
 ```shell
 $ az cloud set --name AzureChinaCloud|AzureGermanCloud|AzureUSGovernment
 ```
 
 ---
 
-Firstly, login to the Azure CLI using:
+1. Log in to the Azure CLI using:
 
-```shell
-$ az login
-```
+    ```shell
+    $ az login
+    ```
 
-~> **NOTE:** Authenticating via the Azure CLI is only supported when using a User Account. If you're using a Service Principal (e.g. via `az login --service-principal`) you should instead [authenticate via the Service Principal directly](authenticating_via_service_principal.html).
+    ~> **NOTE:** Authenticating via the Azure CLI is only supported when using a User Account. If you're using a Service Principal (for example via `az login --service-principal`), you should instead [authenticate via the Service Principal directly](authenticating_via_service_principal.html).
 
-This will prompt you to open a web browser, as shown below:
+1. You are prompted to open a web browser:
 
-```shell
-To sign in, use a web browser to open the page https://aka.ms/devicelogin and enter the code XXXXXXXX to authenticate.
-```
+    ```shell
+    To sign in, use a web browser to open the page https://aka.ms/devicelogin and enter the code XXXXXXXX to authenticate.
+    ```
 
-Once logged in - it's possible to list the Subscriptions associated with the account via:
+1. Once logged in, it's possible to list the Subscriptions associated with the account via:
```

Comment:
does this want to be `3.`?

Reply:
Yes. I "hard-coded" the numbers, which should all display correctly now.